Cargo.lock

```diff
@@ -252,9 +252,9 @@ dependencies = [
  "bitflags 1.3.2",
  "bytes",
  "futures-util",
- "http",
- "http-body",
- "hyper",
+ "http 0.2.11",
+ "http-body 0.4.5",
+ "hyper 0.14.27",
```

Comment:
this means we have a variety of `hyper` versions now, increasing our dependency graph :/ have you checked how many crates we depend on/that need upgrading until we are back to one version?

Reply:
reqwest and igd for released bits.

console-subscriber -> tonic -> axum for dev dependencies.  axum already exists.

Reply:
filed https://github.com/sbstp/rust-igd/issues/58

Reply:
reqwest: https://github.com/seanmonstar/reqwest/issues/2039

Reply:
tonic: https://github.com/hyperium/tonic/issues/1579

Reply:
looks like it will be a while until those libs upgrade :(